src/graph/immutable_graph.cc

```diff
@@ -158,14 +158,32 @@ class HashTableChecker {
   }
 };
 
+ImmutableGraph::EdgeList::Ptr ImmutableGraph::EdgeList::FromCSR(
+    std::vector<int64_t> *indptr, std::vector<dgl_id_t> *indices, std::vector<dgl_id_t> *edge_ids,
+    int sort_on) {
```

Comment:
`sort_on` is a little misleading. It indicates what `indptr` means instead of which endpoint is sorted.